packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -255,6 +256,7 @@ export const MediaLibrary = () => {
                 !assetsLoading && !foldersLoading ? [...folders, ...assets] : []
               }
               selected={selected}
+              sortQuery={query?.sort}
```

Comment:
what happens if this prop is undefined? is it expected to be a `boolean`?

Reply:
if it's undefined it should be an empty string, that's the `defaultProps` I used for it 👀

Reply:
added the fallback directly there too